charts/camunda-platform/templates/connectors/_helpers.tpl

```diff
@@ -16,6 +16,10 @@ If release name contains chart name it will be used as a full name.
   {{- include "zeebe.names.gateway" . | replace "\"" "" -}}:{{- index .Values "zeebe-gateway" "service" "gatewayPort" -}}
 {{- end -}}
 
+{{ define "connectors.operateEndpoint" }}
```

Comment:
What you're describing as `/operate` is called `path`. Essentially, endpoint - is a resource locator. For example, AWS services define `endpoint` as an URL but only because they don't have `path` in their AWS API. [See example](https://docs.aws.amazon.com/general/latest/gr/ct.html).

---

URL on the other hand is not self-descriptive enough. Like, operate URL can be anything. By declaring it an endpoint, we are implicitly referring that this is an API endpoint. To be more explicit I can rename `operateEndpoint` => `operateApiEndpoint`.

Does it make sense to you?

Reply:
On a second thought, I think, `operateURL` makes more sense if we're moving it global helpers.